### PR TITLE
LPS-185518 UpgradeJavaSetResultsSetTotalMethodCheck

### PR DESCRIPTION
**Pull request by TamyrisTorres**
Ticket: https://issues.liferay.com/browse/LPS-185518

Hello Kevin.
This check asks to replace the 'setResults' method with 'setResultsAndTotal'. Also, remove the 'setTotal' method.

Can you review this pull request, please?

### CODE REVIEW

**Comment by TamyrisTorres** [comment]
ci:test:sf

**Comment by TamyrisTorres** [comment]
ci:test:sf

**Comment by TamyrisTorres** [comment]
ci:test:relevant

**Comment by kevhlee** [comment]
Taking a look.

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaSetResultsSetTotalMethodCheck.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+	private String _replaceSetResults(String content) {
+		String newContent = content;
+		Matcher setResultsMatcher = _setResultsPattern.matcher(content);
+
+		while (setResultsMatcher.find()) {
+			String methodCall = JavaSourceUtil.getMethodCall(
+				content, setResultsMatcher.start());
+
+			if (_hasClassName("SearchContainer", content, methodCall)) {
```
Comment (kevhlee) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaSetResultsSetTotalMethodCheck.java:87]: Is specifying the class name necessary? Unless we are dealing with multiple classes, the class name will always be `SearchContainer`.

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaSetResultsSetTotalMethodCheck.testjava [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+/**
+ * @author Tamyris Bernardo
+ */
+public class UpgradeJavaSetResultsSetTotalMethodCheck {
+
+	public void method() {
+
+		SearchContainer searchContainer = new SearchContainer();
+
+
```
Comment (kevhlee) [inline modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaSetResultsSetTotalMethodCheck.testjava:26]: We should remove the newline here.

You can do this by obtaining the method call's indentation using `JavaSourceUtil.getIndent()` and do something like this:

```java
String indent = JavaSourceUtil.getIndent(...);

String fullMethodCall = StringBundler.concat(
    StringPool.NEW_LINE, indent, methodCall, StringPool.SEMICOLON, StringPool.NEW_LINE);

newContent = StringUtil.replace(newContent, fullMethodCall, "");
```

**Review by kevhlee — CHANGES_REQUESTED** [review]

**Pull request closed without merging**